profile/README.md

```diff
@@ -42,4 +49,43 @@ Welcome to the **Kristu Jayanti Google Developer Student Clubs** (KJGDSC) commun
     It is a creative group of individuals passionate about capturing and sharing moments that matter. Led by our visionary leader, we specialize in visual storytelling, graphic design, and multimedia production. 
 
 - ### Design Team
-    It is a collective of creative minds dedicated to crafting visually stunning and user-centric experiences. Led by our talented lead designer, we specialize in graphic design, user interface, and user experience design.
\ No newline at end of file
+    It is a collective of creative minds dedicated to crafting visually stunning and user-centric experiences. Led by our talented lead designer, we specialize in graphic design, user interface, and user experience design.
+
+## What We Do
+At KJC GDSC, we engage in a wide range of activities that reflect our commitment to tech-driven transformation. Here's a glimpse of what we're all about:
+
+- **Open Source Contributions**: Our talented team members actively contribute to open source projects, helping to shape the future of technology.
+
+- **Tech Workshops and Seminars**: We organize workshops and seminars to empower students with knowledge about the latest technologies and industry best practices.
+
+- **Hackathons and Coding Challenges**: We host exciting coding competitions and hackathons that foster innovation and creativity among our members.
+
+## Our Mission
+
+Our mission is to empower students with the knowledge, tools, and resources to become skilled developers and technology enthusiasts. We aim to create a vibrant community that not only excels academically but also actively contributes to the betterment of society.
+
+## Join Our Journey
+We're always on the lookout for like-minded individuals who share our passion for tech, innovation, and community service. Whether you're an experienced coder or just starting your tech journey, there's a place for you here at KJC GDSC. Join our mission to make a difference through technology!
+
+## Get Involved
+
+If you'd like to contribute to our projects, collaborate with our members, or have ideas for initiatives that align with our mission, feel free to reach out to us. You can also check out our [project repository](https://github.com/KJC-GDSC) to see what we're currently working on.
+
+## Our Commitment
+
+KJC GDSC is a student-driven initiative, created by the students, for the betterment of society and our college. We are committed to making a meaningful impact through technology, and we invite you to be a part of this exciting journey.
+
+## Let's Build the Future Together
+
+Our journey at KJC GDSC is an ever-evolving tech odyssey. We invite you to explore our GitHub repositories, contribute, and become a part of our story. Together, we'll continue to code for good, learn together, and innovate for a better tomorrow.
+
+## Connect With Us
+
+- Website: [website](https://developers.google.com/community/gdsc)
```

Comment:
https://gdsc.community.dev/kristu-jayanti-college-bengaluru/

Please add this link. Instead of what is there